torch/csrc/api/src/nn/modules/_functions.cpp

```diff
@@ -0,0 +1,123 @@
+#include <torch/nn/modules/_functions.h>
+
+namespace torch {
+namespace nn {
+namespace functions {
+
+Variable CrossMapLRN2d::forward(
+    AutogradContext *ctx, 
+    Variable input, 
+    const CrossMapLRN2dOptions& options){
+  ctx->saved_data["size"] = options.size();
+  ctx->saved_data["alpha"] = options.alpha();
+  ctx->saved_data["beta"] = options.beta();
+  ctx->saved_data["k"] = options.k();
+  ctx->saved_data["scale"] = torch::Tensor();
+
+  TORCH_CHECK(input.dim() == 4);
+
+  ctx->saved_data["scale"] = torch::empty({0}, torch::TensorOptions().dtype(input.dtype()).device(input.device()))
+                                 .__or__(ctx->saved_data["scale"].toTensor());
```

Comment:
It occurs error 
Expected a Tensor of type Variable but found an undefined Tensor for argument #1 'other'
in __or__ operation